wrappers/src/shared_realm_cs.cpp

```diff
@@ -410,6 +410,16 @@ REALM_EXPORT void shared_realm_write_copy(SharedRealm* realm, uint16_t* path, si
     });
 }
 
+REALM_EXPORT void shared_realm_write_copy_without_client_file_id(SharedRealm* realm, uint16_t* path, size_t path_len, char* encryption_key, bool allow_overwrite, NativeException::Marshallable& ex)
+{
+    handle_errors(ex, [&]() {
+        Utf16StringAccessor pathStr(path, path_len);
+
+        // by definition the key is only allowed to be 64 bytes long, enforced by C# code
+        realm->get()->write_copy_without_client_file_id(pathStr, BinaryData(encryption_key, encryption_key ? 64 : 0), allow_overwrite);
```

Comment:
Why is this exposed as a different method in Core? I would very much prefer that they consolidate `write_copy` and `write_copy_without_client_file_id` as I don't see a point in having two different things here.

Reply:
^ @jedelbo I had the same thought. What's your opinion about having the `copy client file id` part as a parameter to that function instead of having two functions?

Reply:
It shouldn't even be a parameter - there's no reason to ever copy the client file Id.

Reply:
Fixed with the latest commit.